src/App.js

```diff
@@ -1,25 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
-
+import logo from "./logo.svg"
```

Comment:
I think we can take out this line importing the React placeholder logo, but otherwise looks good!